.github/workflows/image_build.yml

```diff
@@ -9,6 +9,8 @@ on: # yamllint disable-line rule:truthy
     branches: [main]
     tags:
       - v*
+  paths-ignore:
+    - docs/**
```

Comment:
The other option is to only rebuild the images in pull requests with changes to this file or changes to one of the Containerfiles:
```
  pull_request:
    paths:
      - .github/workflows/image-build.yml
      - Containerfile.gkm-agent
      - Containerfile.gkm-csi
      - Containerfile.gkm-operator
```

Reply:
Yeah I guess we can do this. I was just thinking that if there's code changes merged then it made sense to rebuild the images with the latest code. Not just on container file updates.

Reply:
The way you have it is fine, it was just a suggestion.